packages/katello/katello/restore.rb

```diff
@@ -144,7 +144,17 @@ def restore_databases
       puts "Logical backup detected, using the standard backup files to restore" if valid_logical_backup
       if @pulp_data_exists
         puts "Restoring Pulp data"
-        run_cmd("tar --selinux --overwrite --listed-incremental=/dev/null -xf pulp_data.tar -C /")
+
+        tar_command = "tar --selinux --overwrite --listed-incremental=/dev/null "
+
+        if File.exist?('pulp_data.tar-2')
```

Comment:
The file currently is not called this, looks like this will need updating.

Reply:
fixed